services/121-service/src/financial-service-providers/financial-service-provider.dto.ts

```diff
@@ -24,4 +24,12 @@ export class FinancialServiceProviderDto {
     ],
   })
   readonly attributes: { name: string; isRequired: boolean }[];
+
+  @ApiProperty({
+    example: [
+      { name: 'columnsToExport', isRequired: true },
+      { name: 'columnToMatch', isRequired: true },
+    ],
+  })
+  readonly configurationProperties: { name: string; isRequired: boolean }[];
```

Comment:
do not include this and other unneeded attributes. (as discussed)

Reply:
I applied these changes to src/program-financial-service-provider-configurations/dtos/program-financial-service-provider-configuration-response.dto.ts, because when calling GET financial-service-providers you do want these attributes